### ISSUE TITLE
kotlintest support

### ISSUE THREAD

**Issue by elect86** [cbeust/kobalt#312]
I'd be nice to have support for [kotlintest](https://github.com/kotlintest/kotlintest) :)

A couple of repos of mine:

- [kotlin-unsigned](https://github.com/elect86/kotlin-unsigned)
- [glm](https://github.com/elect86/glm)

**Comment by cbeust** [comment]
I added support for KotlinTest and it was, as I expected, trivial:

https://github.com/cbeust/kobalt/commit/24174b40f10e1f2166b84d38b58c3a706d3f188d

However, I can't yet compile `kotlin-unsigned` because you're using 1.1-M04 and Kobalt is using 1.06. Mixing these two leads to an "incompatible jar files on the classpath" error.

But I was able to test KotlinTest on a simple Kotlin project and it's working fine.

**Comment by elect86** [comment]
Very nice news, Cedric!

Do you have some timings/deadlines about implementing new Kotlin versions in Kobalt?

**Comment by cbeust** [comment]
I'm looking into moving Kobalt to 1.1 but I'm not sure I want to do this just yet for two reasons:

- It's still an unstable version
- Bumping tp 1.1 means bumping the Kobalt IntelliJ plug-in as well, and this might break users who are choosing to stick with 1.0.6.

**Comment by cbeust** [comment]
I've got everything running with 1.1 (both Kobalt and the Kobalt IDEA plug-in):

```
───── kotlin-unsigned:test
JUnit version 4.12
..[WARN] When comparing doubles consider using tolerance, eg: a shouldBe b plusOrMinus c
.[WARN] When comparing doubles consider using tolerance, eg: a shouldBe b plusOrMinus c
.[WARN] When comparing doubles consider using tolerance, eg: a shouldBe b plusOrMinus c
.[WARN] When comparing doubles consider using tolerance, eg: a shouldBe b plusOrMinus c

Time: 0.059

OK (5 tests)

All tests passed
```

I want to use the IDEA plug-in a bit more before taking the plunge to 1.1, though, but it's looking good so far and it seems to be working fine.

**Comment by elect86** [comment]
Excellent, is that the 0.911 build?

**Comment by cbeust** [comment]
It's a build I haven't released yet since it's Kotlin 1.1 based (if I release it, the IDEA plug-in will stop working until I also update it to 1.1).

**Comment by elect86** [comment]
Ok, I'll keep an eye on the releases then, thanks :)
